noxfile.py

```diff
@@ -0,0 +1,133 @@
+"""
+Nox is a command-line tool that automates testing in multiple Python environments, similar to tox.
+Unlike tox, Nox uses a standard Python file for configuration.
+
+To run all sessions defined in the noxfile.py file, run the following command:
+$ nox
+
+To run a specific session, run the following command:
+$ nox -s <session-name>
+
+To run a session with additional arguments, run the following command:
+$ nox -s <session-name> -- <additional-arguments>
+
+To list all available sessions, run the following command:
+$ nox -l
+"""
+
+import os
+
+import nox
+from nox.sessions import Session
+
+# default sessions (sorted alphabetically)
+nox.options.sessions = ["bandit", "black", "flake8", "isort", "mypy", "unit-tests"]
+
+# reuse virtual environment across all sessions
+nox.options.reuse_venv = "always"
+
+# stop other sessions if one session fails
+nox.options.stop_on_first_error = True
```

Comment:
Have we reached a consensus on this one? (It seemed to me that I was the only one in favor of this option).

Reply:
After finishing [this](https://github.com/containerbuildsystem/cachi2/pull/680) I think the execution of all sessions is quite fast now.

Reply:
Like I expressed elsewhere in a thread, I'm not so much in favour of an early failure here given that the bottleneck are always UTs no matter what order this is executed in.